charts/metallb/templates/prometheusrules.yaml

```diff
@@ -19,8 +19,8 @@ spec:
     {{- if .Values.prometheus.prometheusRule.staleConfig.enabled }}
     - alert: MetalLBStaleConfig
       annotations:
-        message: {{`'{{ $labels.job }} - MetalLB {{ $labels.container }} on {{ $labels.pod
-          }} has a stale config for > 1 minute'`}}
+        summary: {{`'Stale config on {{ $labels.pod }}'`}}
```

Comment:
Will this change affect users monitoring dashboards? I think not, because it'd still be an alert and this is the metadata attached to it, but double checking